cmd/argo/commands/delete.go

```diff
@@ -36,7 +36,7 @@ func NewDeleteCommand() *cobra.Command {
   argo delete @latest
 `,
 		Run: func(cmd *cobra.Command, args []string) {
-			if len(args) == 0 && !(all || allNamespaces || flags.completed || flags.resubmitted || flags.prefix != "" || flags.labels != "" || flags.fields != "" || flags.finishedAfter != "") {
+			if len(args) == 0 && !(all || allNamespaces || flags.completed || flags.resubmitted || flags.pending || flags.prefix != "" || flags.labels != "" || flags.fields != "" || flags.finishedAfter != "") {
 				cmd.HelpFunc()(cmd, args)
 				os.Exit(1)
 			}
```

Comment:
[line 55](https://github.com/argoproj/argo-workflows/blob/28ffd1c442260dc8ef222062c41d2c33e880bd65/cmd/argo/commands/delete.go#L55) below would also need modifications

Reply:
added. thanks for pointing it out.